src/main/java/blockchains/iaas/uni/stuttgart/de/plugin/ethereum/AbstractAdapter.java

```diff
@@ -0,0 +1,26 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Institute for the Architecture of Application System - University of Stuttgart
+ * Author: Ghareeb Falazi
+ *
+ * This program and the accompanying materials are made available under the
+ * terms the Apache Software License 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *******************************************************************************/
+package blockchains.iaas.uni.stuttgart.de.plugin.ethereum;
+
+import blockchains.iaas.uni.stuttgart.de.api.interfaces.BlockchainAdapter;
+import blockchains.iaas.uni.stuttgart.de.api.interfaces.FinalityConfidenceCalculator;
+
+public abstract class AbstractAdapter implements BlockchainAdapter {
```

Comment:
Similar to the Bitcoin case, this class is not useful for one adapter only. Therefore, please remove it and add its contents to the `EthereumAdapter` class